packages/ui/src/components/sheet/components/sheet-header.native.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, QuestionCircleIcon, Text, TouchableOpacity } from '../../../../native';
+
+const SHEET_HEADER_HEIGHT = 72;
+
+export function SheetHeader({
+  title,
+  icon,
+  onPressSupport,
+}: {
+  title: string;
+  icon: React.ReactNode;
+  onPressSupport?(): unknown;
+}) {
```

Comment:
Can we create a separate interface for the props, [re convention here](https://www.notion.so/trustmachines/Naming-Component-Props-13e6ba7dd5ab4404a1cda0de6175cda6?pvs=4)